examples/sample-cmd/main.go

```diff
@@ -23,6 +25,43 @@ func main() {
 		return fmt.Sprintf("Hello %s!", c.Param("name")), nil
 	})
 
+	app.SubCommand("spinner", func(ctx *gofr.Context) (interface{}, error) {
+		// initialize the spinner
+		spinner := terminal.NewDotSpinner(ctx.Out).Spin(ctx).Spin(ctx)
```

Comment:
Spin().Spin() ?

The methods names could be confusing

Reply:
I wanted to chain the Stop method too, cause doing that we can write one defer command to start and stop the spinner.

I can return nil from the Spin() which starts the spinner.